release/create-mxcheckouts.py

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
+"""Parse checkouts from a buildout configuration and write to a mxdev
+compatible ini file.
+
+This script is called by 'tox -c release/tox.ini -e create-mxcheckouts.py'
+
+Manual usage: bin/python create-mxcheckouts.py
+"""
+
+# From: create-constraints.py
+
+import os
+import sys
+
+from zc.buildout import buildout
+
+if not len(sys.argv) == 3:
+    print("ERROR. Usage: create-mxcheckouts.py buildout.cfg checkouts.ini")
+    sys.exit(1)
+config_file = sys.argv[1]
+config_file = os.path.realpath(config_file)
```

Comment:
Using pathlib.Path makes it all much nicer to read 😃

Sorry on the phone ,I van provide suggestions later 😃